app/views/spree/api/payments/source_views/_bolt.json.jbuilder

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
 json.call(payment_source, :id, :created_at, :card_token, :card_last4, :card_bin, :card_number, :card_expiration,
-  :card_postal_code, :card_id, :create_bolt_account)
```

Comment:
I don't think we should expose the `user_id`

Reply:
fixed in a638e20764ad94df53900934df0e398450960ae2

Reply:
How can I find that commit? Usually, we reply with the URL of the commit, rather than just the hash.